### PR TITLE
Why not a MOOC?

### PR DESCRIPTION
**Pull request by gvwilson**
Adding material to the main lesson.

### CODE REVIEW

File: _episodes/16-practices.md [hunk 1]
```diff
@@ -228,6 +228,40 @@ Peak rule.
...
+> In addition,
+> most people are much less comfortable recording themselves
+> than contributing written material.
+>
+> The second answer is that doing significantly outperforms watching.
+> A [recent paper by Koedinger et al]({{ site.github.url }}/files/papers/koedinger-doing-watching-2015.pdf)
+> estimated "...the learning benefit from extra doing (1 SD increase)
+> to be more than six times that of extra watching or reading."
+> And while we do not (yet) have empirical data,
+> we believe very strongly that many noviceswould give up in despair
```
Comment (abought) [inline _episodes/16-practices.md]: Tiny note- there seems to be a missing space: 
`novices would`

Comment (gvwilson) [inline _episodes/16-practices.md]: Thanks - corrected in 8b5ca4b

File: _episodes/16-practices.md [hunk 2]
```diff
@@ -228,6 +228,40 @@ Peak rule.
...
+> Making a small change to a text file like this one only takes a few minutes.
+> but making *any* change to a video takes an hour or more.
+> In addition,
+> most people are much less comfortable recording themselves
+> than contributing written material.
+>
+> The second answer is that doing significantly outperforms watching.
+> A [recent paper by Koedinger et al]({{ site.github.url }}/files/papers/koedinger-doing-watching-2015.pdf)
+> estimated "...the learning benefit from extra doing (1 SD increase)
+> to be more than six times that of extra watching or reading."
```
Comment (davis68) [inline _episodes/16-practices.md]: There's a lot going on in that paper.  Suggested clarification along these lines:

    The second answer is that doing significantly outperforms watching.  Specifically, a [recent paper by Koedinger et al]({{ site.github.url }}/files/papers/koedinger-doing-watching-2015.pdf) estimated "...the learning benefit from extra doing (1 SD increase) to be more than six times that of extra watching or reading."  *Doing*, in this case, refers to completing an interactive or mimetic task with feedback, while *benefit* refers to both *completion rates* and *overall performance*.

File: _episodes/16-practices.md [hunk 3]
```diff
@@ -228,6 +228,40 @@ Peak rule.
...
+>
+> Many learners find it difficult to get to a workshop,
+> either because there isn't one locally
+> or because it's difficult to schedule time around other commitments,
+> so why don't we create video recordings of the lessons
+> and offer the workshop as a MOOC (Massive Open Online Course)?
+>
+> The first answer is that we did in 2010-11,
+> but found the maintenance costs unsustainable.
+> Making a small change to a text file like this one only takes a few minutes.
```
Comment (lexnederbragt) [inline _episodes/16-practices.md]: 'a text file this one' --> 'this webpage'

**Comment by davis68** [comment]
That felt hacky.

But it looks like it [worked](https://swcarpentry.github.io/instructor-training/15-practices/).

**Pull request merged**